File: src/main/java/net/ccbluex/liquidbounce/utils/animations/Direction.java
package net.ccbluex.liquidbounce.utils.animations;
public enum Direction {
    FORWARDS,
    BACKWARDS;

    public Direction opposite() {
        if (this == Direction.FORWARDS) {
            return Direction.BACKWARDS;
        } else return Direction.FORWARDS;
    }


    public boolean forwards() {
        return this == Direction.FORWARDS;
    }

    public boolean backwards() {
        return this == Direction.BACKWARDS;
    }

}

File: src/main/java/net/ccbluex/liquidbounce/utils/animations/Direction.kt
/*
 * FDPClient Hacked Client
 * A free open source mixin-based injection hacked client for Minecraft using Minecraft Forge.
 * https://github.com/SkidderMC/FDPClient/
 */
package net.ccbluex.liquidbounce.utils.animations

enum class Direction {
    FORWARDS,
    BACKWARDS;

    fun opposite(): Direction = if (this == FORWARDS) BACKWARDS else FORWARDS
}
